### PR TITLE
Golang 112

### PR DESCRIPTION
**Pull request by Toasterson**
I needed to get rid of all binaries in the testdata as they kept failing pkg's vaildations. Disabling the checks did not work.

### CODE REVIEW

File: components/developer/golang-112/golang.p5m [hunk 1]
```diff
@@ -23,4 +23,4 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-depend type=require fmri=developer/golang-111
+depend type=require fmri=developer/golang-112
```
Comment (Mno-hime) [inline components/developer/golang-112/golang.p5m:26]: Why this? Perhaps you just wanted to `cp` the file and you `git mv` it?

**Review by Mno-hime — COMMENTED** [review]
Thanks for doing this!

Maybe it's also time to ditch some of the old, unsupported golangs? Namely 1.8 and 1.10.

**Comment by Toasterson** [comment]
We can drop 1.8 and 1.10

File: components/developer/golang-112/golang.p5m (hunk 1 above)
Comment (Toasterson) [reply components/developer/golang-112/golang.p5m:26]: i wanted to set golang 1.12 as the default thus i moved the golang package manifest to it's new folder like back when i added go 1.10

**Comment by Mno-hime** [comment]
Did you rebuild sample manifest? I get several updates in that file and they should be possibly added to the golang-112.p5m file: [sample-manifest.diff.txt](https://github.com/OpenIndiana/oi-userland/files/3587036/sample-manifest.diff.txt).

**Comment by Toasterson** [comment]
I did but I had problems with it. I'll try to get the same result when rebuilding. sample-manifest did at first not like my locale. I'll see what i can do.

**Review by Mno-hime — APPROVED** [review]

**Comment by alarcher** [comment]
Argh... again same issue merging on the phone... did not see that it was not squashed...

**Pull request merged**